engine/dlib/src/dlib/buffer.cpp

```diff
@@ -616,4 +644,117 @@ namespace dmBuffer
         buffer->m_ContentVersion++;
         return RESULT_OK;
     }
+
+    // Returns a pointer to the metadata item or 0 if not found.  Assumes buffer is valid.
+    Buffer::MetaData* FindMetaDataItem(Buffer* buffer, dmhash_t name_hash) {
```

Comment:
We favor adding "static" to private functions to keep implementation details private, and allow for translation unit optimizations.

Also, it's for communication purposes as it makes the intent and scope clearer for the reader.

Reply:
Ok, clear  :+1: